src/test/java/com/revature/services/ProductServiceTest.java

```diff
@@ -79,6 +80,18 @@ void testFindById() {
 		assertEquals(expected, actual);
 		verify(this.mockProductRepo, times(1)).findById(id);
 	}
+	
+	@Test
+	void testFindById_Failure_ProductNotFound() {
+		int id = 404;
+		given(this.mockProductRepo.findById(id)).willReturn(Optional.empty());
+		try {
+			this.pServ.findById(id);
+			fail("Expected an exception to be throw");
```

Comment:
You should specify which exception type is expected to be thrown. In this case, it's `ProductNotFoundException`